File: snippets/python/get_the_results_of_a_batch_operation.py
import urllib.request

response_body_url = "RESPONSE_BODY_URL_FROM_PREVIOUS_STEPS"
data = ""
with urllib.request.urlopen(response_body_url) as response:
   data = response.read()

# Extract data from gzipped archive, return as JSON array
results = process_batch_archive(data)
for result in results:
    user = fakeDB.findUser(result["operation_id"])
    fakeDB.updateUser(user, {
        "mailchimp_web_id": result["response"]["web_id"]
    })

File: snippets/node/get_the_results_of_a_batch_operation.js
const fetch = require("node-fetch");

const responseBodyUrl = "RESPONSE_BODY_URL_FROM_PREVIOUS_STEPS";

async function run() {
  const response = await fetch(responseBodyUrl);

  // Extract data from gzipped archive, return as JSON array
  // Implementation details not included
  const results = processBatchArchive(response);
  results.forEach(result => {
    const user = fakeDB.findUser(result.operation_id);
    fakeDb.updateUser(user, {
      mailchimpWebId: result.response.web_id
    });
  });
}

run();